docs/manual/service_catalog.md

```diff
@@ -101,6 +101,8 @@ The playbook will:
 
 Day 2 operations are operations that **update** or **delete** existing resources.
 
+> **Note: ** If you want to be able to specify extra_vars on a schedule, you must select Prompt on Launch for EXTRA VARIABLES on the workflow template, or a enable a survey on the workflow template, then those answered survey questions become extra_vars.
```

Comment:
To be move at line 31.

By default, recent version of AWX/Tower drop extra variables that are not declared in the survey. To be able to receive Squest extra vars you need to enable "Prompt on Launch" in the "Variables" section of you job template. This correspond to the flag "ask_variables_on_launch" of the job_template model on the Tower/AWX API.